WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -205,6 +204,17 @@ protected void onStart() {
 
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onSiteRemoved(OnSiteRemoved event) {
+        if (!event.isError()) {
+            getAdapter().loadSites();
+        } else {
+            // shouldn't happen
```

Comment:
I think a log is useful, good idea 👍 

I'm wondering if we should localize the error string, but since we don't expect it to occur I'm leaning towards no...Thoughts?